components/sysutils/net-snmp/history

```diff
@@ -1,2 +1 @@
-library/python-2/net-snmp-26@5.4.4,5.11-2016.0.1.0
-library/python-2/net-snmp-27@5.4.4,5.11-2016.0.1.0 library/python/net-snmp-27
+system/management/snmp/net-snmp/documentation@5.7.3-2018.0.0.4 system/management/snmp/net-snmp/demo
```

Comment:
Why have you removed old content?

Reply:
I have to admit that I don't understand this file. 
I guess the added line is unnecessary too.
Drop the changes in history?

Please have a look at the end of net-snmp-demo.p5m.
Actually it is:
depend type=optional \
    fmri=system/management/snmp/net-snmp/documentation@5.7.3,5.11-11.5.0.0.0.50.0
Should this be:
depend type=optional \
    fmri=system/management/snmp/net-snmp/documentation@5.7.3

Reply:
It should be the version of package with rename attribute (which is generated based on history file, look at https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/make-rules/ips.mk#L329 )